### DISCUSSION TITLE
How do we keep communities safe for younger audiences?

### DISCUSSION THREAD

**Discussion by Akshu-on-github** [EddieHubCommunity/support#605] [General] [answered]
We had this conversation on discord, but let's keep it open for discussion. Many including @stemount and @nhcarrigan put forward some really nice points on #chatting, let's bring the same over here!

**Comment by eddiejaoude** [comment]
This is a great and difficult topic 👍 - @Akshu-on-github can you copy and paste some of the conversation that happened on Discord?

I personally like to keep our communities safe for everyone no matter what age, but I know that is easier said than done. I would be interested to hear what other people think, have seen and have tried 🤓

**Comment by Akshu-on-github** [comment, marked as answer]
Pasting a few key messages below to provide a summary. Please note that the questions are paraphrased and only the key parts of the messages are quoted.

**TL;DR**: A discussion on safe spaces for younger teens in coding communities.

### I consider minors as children. However, opensource ideals say that we shouldn't treat people differently based on, among other things, age. What are your thoughts on this?

> Regardless of age, I give my contributors the level of support/assistance they need - and try to keep my feedback both positive and constructive.
@nhcarrigan 

> Age is just a number. skills and passion matters...if you have passion about something and you have skills to achieve it (...)
@Nitesh-thapliyal
<br>

### But how can we be sure that opensource communities are child-friendly?

> I'm sure that "we shouldn't treat people differently based on, among other things, age." also comes along with "do not send things that are potentially not harmful or nsfw for a community, try not to share sensitive info"
@Vyvy-vi 

> Within my communities, I specifically disallow content that would not be appropriate for minors (...)
For example, if a user shares their email address, I remove it with a response along the lines of "I've removed your message because it contains private info. In general, it is not a safe practise to share personally identifiable information in a public space. While we work to keep our community a safe space, we cannot guarantee the well-intentions of every member at any given time."
@nhcarrigan
<br>

### Strict regulations are feasible in small communities. But what about large ones?

> I'll use freeCodeCamp as an example for that one.
We allow the occasional profanity, such as "Sh*t I just broke my main branch" and respond with a reminder to keep language appropriate. But for excessive profanity, profanity directed at other users, or significantly offensive words, we address with a moderation action.
@nhcarrigan

> I think and would hope most places try to be a safe space for everyone (...)
Setting rules and enforcing them from day 1 with 1 member is just as important as with 100k members - people will follow the community flow - yes some people will push the boundaries and these people are at risk if enforced not the community
@eddiejaoude 

> I think this really is a huge undertaking. Let's say, for example, you do one of a few things:
>  1. You mod a server, but you have two sets of channels. When you first join, you can only access a PG copy of the server that is suitable for younger audiences. (I haven't seen a good example of this yet)
>  2. You have a smaller server (which makes discovery hard) that has MUCH stricter rules. This hurts retention because you'll end up banning users who are useful but do something slightly out of line. (This is typical)
>  3. Even as you do either of these things, you must realise that communities for younger audiences still contain a huge amount of toxicity (I see this every day on another community I mod)
(paraphrased) @stemount

### Anything you'd like to say to the guardians of young opensourcers who are worried about this?

> My honest recommendation: The internet is a big and scary place. And not everything is going to be safe or friendly or welcoming. 
(...) I'd recommend spending a bit of time identifying areas (servers, forums, projects, whatever) that are more safe and considerate for minor users, and ( join ) those communities specifically.
Unfortunately, it's impossible to completely avoid the Bad Things™️, but there are definitely communities that are more friendly for this type of thing.
@nhcarrigan

**Reply by eddiejaoude** [reply]
Thanks for summarising 👍

**Reply by stemount** [reply]
I think @nhcarrigan is on the money here with the internet is big and scary and all protections you can provide to young people are inherently flawed.

Schools, Colleges and similar should be providing these safer(?) spaces within their domain (e.g. pupils at a school/college and their tutors)
